audio_builder.py

```diff
@@ -1,177 +1,245 @@
+CONFIG = configparser.ConfigParser()
+CONFIG.read('config.ini')
+BATCH_CONFIG = configparser.ConfigParser()
+BATCH_CONFIG.read('batch.ini')
+CLOUD_CONFIG = configparser.ConfigParser()
+CLOUD_CONFIG.read('cloud_service_settings.ini')
 
 # Get variables from configs
-nativeSampleRate = int(config['SETTINGS']['synth_sample_rate'])
-originalVideoFile = os.path.abspath(batchConfig['SETTINGS']['original_video_file_path'].strip("\""))
-skipSynthesize = parseBool(config['SETTINGS']['skip_synthesize'])
-forceTwoPassStretch = parseBool(config['SETTINGS']['force_stretch_with_twopass'])
-outputFormat = config['SETTINGS']['output_format'].lower()
-batchSynthesize = parseBool(cloudConfig['CLOUD']['batch_tts_synthesize'])
-tts_service = cloudConfig['CLOUD']['tts_service']
-
-def trim_clip(inputSound):
-    trim_leading_silence: AudioSegment = lambda x: x[detect_leading_silence(x) :]
-    trim_trailing_silence: AudioSegment = lambda x: trim_leading_silence(x.reverse()).reverse()
-    strip_silence: AudioSegment = lambda x: trim_trailing_silence(trim_leading_silence(x))
-    strippedSound = strip_silence(inputSound)
-    return strippedSound
-
-# Function to insert audio into canvas at specific point
-def insert_audio(canvas, audioToOverlay, startTimeMs):
-    # Create a copy of the canvas
-    canvasCopy = canvas
-    # Overlay the audio onto the copy
-    canvasCopy = canvasCopy.overlay(audioToOverlay, position=int(startTimeMs))
-    # Return the copy
-    return canvasCopy
-
-# Function to create a canvas of a specific duration in miliseconds
-def create_canvas(canvasDuration, frame_rate=nativeSampleRate):
-    canvas = AudioSegment.silent(duration=canvasDuration, frame_rate=frame_rate)
-    return canvas
-
-def get_speed_factor(subsDict, trimmedAudio, desiredDuration, num):
-    virtualTempFile = AudioSegment.from_file(trimmedAudio, format="wav")
-    rawDuration = virtualTempFile.duration_seconds
-    trimmedAudio.seek(0) # This MUST be done to reset the file pointer to the start of the file, otherwise will get errors next time try to access the virtual files
+NATIVE_SAMPLE_RATE = int(CONFIG['SETTINGS']['synth_sample_rate'])
+ORIGINAL_VIDEO_FILE = os.path.abspath(
+    BATCH_CONFIG['SETTINGS']['original_video_file_path'].strip("\""))
+SKIP_SYNTHESIZE = parse_bool(CONFIG['SETTINGS']['skip_synthesize'])
+FORCE_STRETCH = parse_bool(CONFIG['SETTINGS']['force_stretch_with_twopass'])
+OUTPUT_FORMAT = CONFIG['SETTINGS']['output_format'].lower()
+BATCH_SYNTHESIZE = parse_bool(CLOUD_CONFIG['CLOUD']['batch_tts_synthesize'])
+TTS_SERVICE = CLOUD_CONFIG['CLOUD']['tts_service']
+
+
+def trim_clip(input_sound: AudioSegment) -> AudioSegment:
+    """Trims leading and trailing silence from audio clip
+
+    Args:
+        input_sound (AudioSegment): Input audio clip
+
+    Returns:
+        AudioSegment: Input audio clip with leading and trailing silence removed
+    """
+    trim_leading_silence: AudioSegment = lambda x: x[detect_leading_silence(
```

Comment:
I don't feel like breaking this into multiple lines improves readability, if anything it feels worse.

Reply:
What exactly? In this particular segment most of the work was renaming the global vars (uppercase) and removed a few lines, in particular not needed assignments. And added docstrings.